Fix build for the AWS Kinesis exporting connector (#9823)
netdata/netdata: exporting/aws_kinesis/README.md

## Prerequisites

To use AWS Kinesis for metric collecting and processing, you should first
[install](https://docs.aws.amazon.com/en_us/sdk-for-cpp/v1/developer-guide/setup.html) AWS SDK for C++.
`libcrypto`, `libssl`, and `libcurl` are also required to compile Netdata with Kinesis support enabled. Next, Netdata
should be re-installed from the source. The installer will detect that the required libraries are now available.

> If the AWS SDK for C++ is being installed from source, it is useful to set `-DBUILD_ONLY=kinesis`. Otherwise, the

Reviewer: ```suggestion
If you're installing the AWS SDK for C++ from source, it is useful to set `-DBUILD_ONLY=kinesis`. Otherwise, the
```

> build process could take a very long time. Note, that the default installation path for the libraries is

Reviewer: ```suggestion
build process could take a very long time. The default installation path for the libraries is
```

`/usr/local/lib64`. Many Linux distributions don't include this path as the default one for a library search, so it is
advisable to use the following options to `cmake` while building the AWS SDK:

```sh
sudo cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_ONLY=kinesis <aws-sdk-cpp sources>
```

The `-DCMAKE_INSTALL_PREFIX=/usr` option also ensures that

> [third party dependencies](https://github.com/aws/aws-sdk-cpp#third-party-dependencies) are installed in your system

Reviewer: ```suggestion
[third-party dependencies](https://github.com/aws/aws-sdk-cpp#third-party-dependencies) are installed in your system
```
